src/components/NavBar/NavBar.styled.js

```diff
@@ -0,0 +1,39 @@
+import styled from "styled-components";
+
+export const Header = styled.header`
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    box-shadow: 0px 0px 15px black;
+    margin-bottom: 30px;
+`;
+
+export const Logo = styled.a`
+    margin-left: 30px;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    @media screen and (max-width: 500px) {
```

Comment:
Sizes for responsive could be also moved to global variables

Reply:
Created global variables